control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -55,8 +59,9 @@ import (
 
 const (
 	// TopicPrefix is the Kafka Channel topic prefix - (topic name: knative-messaging-kafka.<channel-namespace>.<channel-name>).
-	TopicPrefix          = "knative-messaging-kafka"
-	DefaultDeliveryOrder = contract.DeliveryOrder_ORDERED
+	TopicPrefix                     = "knative-messaging-kafka"
+	DefaultDeliveryOrder            = contract.DeliveryOrder_ORDERED
+	NewChannelDispatcherServiceName = "kafka-channel-dispatcher"
```

Comment:
This should be `kafka-channel-ingress`.

https://github.com/knative-sandbox/eventing-kafka-broker/blob/fc34bb79aeec648be20cd0df2ea0c21fc7b1d217/data-plane/config/channel/500-receiver.yaml#L164

Reply:
I wouldn't use dispatcher in this constant name since it might be confusing to what we're referring to, dispatcher meant a different component before.